### DISCUSSION TITLE
Category not automatically selected on New Discussion

### DISCUSSION THREAD

**Discussion by SachinShekhar** [community/community#3037] [Discussions]
When hitting `New Discussion` button from a category, I expect that category is automatically selected. But, that doesn't happen.

This is important when redirecting users from Issues. As a category can be open-ended or answer-based, wrong category can be disastrous.

**Comment by evi-liu** [comment]
Thank you for the feedback! We're following along this thread and gauging customer impact via upvotes and reactions.
